src/main/java/ejercicio/banco/repository/MySqlBankRepository.java

```diff
@@ -54,21 +50,21 @@ public Bank find(int id) {
                     String name = rs.getString("name");
                     String address = rs.getString("address");
                     Bank bank = new Bank(id, name, address);
-                    closeConections(con);
+                    DataBaseUtil.closeConections(con);
                     return bank;
                 }
             }
-            closeConections(con);
         } catch (Exception e) {
             e.printStackTrace();
         }
+        DataBaseUtil.closeConections(con);
```

Comment:
Same here. Be careful with data base connections (not sure what would happen if you try to close it twice...)

Reply:
Same as above